### PR TITLE
[FEATURE] [MS-94] Changes in Back to show Job Opportunities at Home Screen

### PR DESCRIPTION
**Pull request by juanandres7698**

### CODE REVIEW

File: routes/public/open-position.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  validateFormat,
+  openPositions.add,
+);
+router.put(
+  '/:id',
+  validateFormat,
+  validateLength,
+  openPositions.edit,
+);
+router.delete('/:id', validateFormat, openPositions.remove);
```
Comment (EugenioPonzone) [inline routes/public/open-position.js:27]: These arent necessary

File: controllers/public/open-positions.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+const remove = (req, res) => {
+  const { id } = req.params;
+  OpenPosition.findByIdAndRemove(id)
+    .then((data) => {
+      if (!data) return res.status(404).json({ msg: `${notFoundTxt} ID: ${id}`, error: true });
+      return res.status(200).json({ msg: 'Position deleted', data });
+    })
+    .catch((err) => res.status(500).json({ msg: `Error: ${err}`, error: true }));
+};
```
Comment (EugenioPonzone) [inline controllers/public/open-positions.js:91]: These arent necessary

File: controllers/public/open-positions.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+module.exports = {
+  getAll,
+  getById,
+  search,
+  add,
+  edit,
+  remove,
```
Comment (EugenioPonzone) [inline controllers/public/open-positions.js:99]: These arent too

File: routes/index.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ const express = require('express');
 const path = require('path');
 const auth = require('./auth');
 
+// const public = require('./');
```
Comment (EugenioPonzone) [inline routes/index.js:6]: const public = require('./public');

File: routes/index.js [hunk 5]
```diff
@@ -20,6 +21,7 @@ router.get('/server-status', (req, res) => res.send({ status: 'Server OK' }));
 
 router.use('/auth', auth);
 
+// router.use('/', authMiddleware, public);
```
Comment (EugenioPonzone) [inline routes/index.js:24]: router.use('/', public);

**Review by BaltazarCortina — APPROVED** [review]

**Review by Sabrucita — APPROVED** [review]

**Pull request merged**